src/components/Column.astro

```diff
@@ -0,0 +1,9 @@
+
+---
+const { sticky = false } = Astro.props;
+const className = sticky ? '[&>:first-child]:mt-0 [&>:last-child]:mb-0 xl:sticky xl:top-24' : '[&>:first-child]:mt-0 [&>:last-child]:mb-0';
```

Comment:
Is there a better way to do this ?

Reply:
@AndyClifford yes you can just have a ternary operator inline:
```
:class="sticky ? '[&>:first-child]:mt-0 [&>:last-child]:mb-0 xl:sticky xl:top-24' : '[&>:first-child]:mt-0 [&>:last-child]:mb-0'"
```

Reply:
Is there a way that we don't have to have such a long line of code?

Reply:
Cheers - seems like astro has slightly different syntax to vue but got it working with a parameterized string